.travis.yml

```diff
@@ -5,25 +5,52 @@ cache:
     - $HOME/.composer/cache
 
 php:
-    - 5.5
-    - 5.6
-    - 7.0
+    - "5.5"
+    - "5.6"
+    - "7.0"
+    - "7.1"
+    - nightly
     - hhvm
 
 matrix:
     fast_finish: true
     include:
-        - php: 7.0
-          env: SYMFONY_VERSION=2.8.*
-        - php: 7.0
-          env: SYMFONY_VERSION=3.0.*
-        - php: 7.0
+        - php: "5.5"
+          env:
+              - "SYMFONY_VERSION=~2.8.0"
```

Comment:
we could just test the lower and upper bound yeah, depends on what @dunglas want, I just continued what was started